moveit_core/collision_detection/src/world.cpp

```diff
@@ -239,6 +239,27 @@ bool World::moveShapeInObject(const std::string& object_id, const shapes::ShapeC
   return false;
 }
 
+bool World::moveShapeInObject(const std::string& object_id, const EigenSTL::vector_Isometry3d& shape_poses)
```

Comment:
_All_ shapes are moved:
```suggestion
bool World::moveShapesInObject(const std::string& object_id, const EigenSTL::vector_Isometry3d& shape_poses)
```

Reply:
Done